sklearn/linear_model/stochastic_gradient.py

```diff
@@ -1256,7 +1253,6 @@ class SGDRegressor(BaseSGDRegressor):
         The learning rate schedule:
 
         - 'constant': eta = eta0
-        - 'optimal': eta = 1.0 / (alpha * (t + t0)) [default]
```

Comment:
Only move `[default]` to the invscaling line in SGDRegressor.

Reply:
Thanx @amueller and  @lesteve. 
Yes @lesteve, I will take care of PR description. 
Made changes as you suggested.